android/build.gradle

```diff
@@ -15,14 +15,14 @@ buildscript {
 }
 
 android {
-    compileSdkVersion 31
-    buildToolsVersion '30.0.3'
+    compileSdkVersion 33
+    buildToolsVersion '33.0.0'
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 31
-        versionCode 93
-        versionName "0.9.3"
+        minSdkVersion 19
+        targetSdkVersion 33
+        versionCode 95
+        versionName "0.9.5"
```

Comment:
@shivamsharma2710 We should update this to version 1.0.0

Reply:
this is done. please check.